source/agora/common/API.d

```diff
@@ -90,4 +90,22 @@ public interface API
     ***************************************************************************/
 
     public PublicConfig getPublicConfig();
+
+    /***************************************************************************
+
+        Send message to this node
+
+        Parameter: 
+            string message(Hex)
```

Comment:
If you want to pass a hash around, you could just use `Hash` from `scpd.types.Stellar_types`. For example:

```D
public void setMessage(Hash hash)
```